File: DTO/Models/Command.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DTO.Models
{
    public class Command
    {
        public int Id { get; private set; }
        public int IdAddress { get; private set; }
        public int IdClient { get; private set; }
        public DateTime? DateOfCommand { get; set; }
        public List<CommandWine> CommandWine { get; set; }
        public Command(int id, int idAddress, int idClient, List<CommandWine> commandWine, DateTime? dateOfCommand = null)
        {
            Id = id;
            IdAddress = idAddress;
            IdClient = idClient;
            CommandWine = commandWine;
            DateOfCommand = dateOfCommand;
        }
    }
}

File: Angular-Project/WineSellingProject/src/app/Models/Command.ts
import { CommandWine } from "./CommandWine";

export class Command {
    id: number;
    idClient: number;
    idAddress: number;
    CommandWine: CommandWine[];
    
    constructor(Id: number, IdClient: number, IdAddress: number, CommandWine : CommandWine[]) {
        this.id = Id;
        this.idClient = IdClient;
        this.idAddress = IdAddress;
        this.CommandWine = CommandWine;
    }
}
